tests/end2end/playwright/form_relational_value.spec.js

```diff
@@ -0,0 +1,52 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+
+test.describe('Form relational values', () => {
+
+    test.beforeEach(async ({ page }) => {
+        const url = '/index.php/view/map/?repository=testsrepository&project=form_type_relational_value';
+        await page.goto(url, { waitUntil: 'networkidle' });
+
+        await page.locator('#button-edition').click();
+        await page.locator('#edition-draw').click();
+    });
+
+    test('Checkboxes relational values', async ({ page }) => {
+        const groups = [
+            { groupName: "test", count: 3, expectNot: true },
```

Comment:
```suggestion
            { groupName: "test", count: 3, expect: false },
```
Maybe easier to read and understand?